travis/licenses_golden/licenses_flutter

```diff
@@ -1062,6 +1062,8 @@ ORIGIN: ../../../LICENSE
 TYPE: LicenseType.bsd
 FILE: ../../../flutter/.travis.yml
 FILE: ../../../flutter/DEPS
+FILE: ../../../flutter/frontend_server/bin/starter.dart
+FILE: ../../../flutter/frontend_server/lib/server.dart
```

Comment:
Does this code end up in any binaries we ship? If not, maybe we should just exclude it from the licenses script.